### DISCUSSION TITLE
Group 8: Credit Builder Loans

### DISCUSSION THREAD

**Discussion by bbdaniels** [gui2de/ppol768-spring23#164] [Group Projects Feedback]
Group 8: Please add a short title we can use to refer to your project! I like the area of interest, but in general, this project needs a fair amount of additional detail: I describe two main questions below.

The biggest question here is around the construction of comparison groups. Who are the eligible and ineligible people -- and where are you recruiting these pools from? I am having trouble understanding the idea of "before/after" the program, because I don't see a group who doesn't have access to the program. A regression discontinuity approach here would ideally look something like the following: "GASFCU created a program in 20?? that allowed members with credit scores lower than XXX to participate in a new CBL program. We will compare financial outcomes for people just below the credit score threshold with those just above the threshold before and after the availability of the CBL program to investigate long-term effects of CBL borrowing on financial health and credit scores in the long term." Since that doesn't appear to be the design of the program (unless I'm misunderstanding), you need to be rather more specific about who are treatment and control groups here.

You should further be more explicit with approaches to take-up. From the write-up here, I understand that everyone at the CU got access to the same program at the same time, so as above, who forms your control group? It certainly cannot be people who do not _choose_ to use the features, as they are likely to be fundamentally different from those who do. I am confused here because your subjects and comparison groups are written exactly the same in the Methodology.

For data, can you elaborate a bit more on outcomes? If your intention is to _solely_ use administrative data from the credit union, what is your ability to look at overall financial outcomes for people (who may have other accounts or family members)? Why are you limiting your sample to 200 people when you could have administrative data access to all CU members? You should feel free to do something like combine financial data (which is a great idea) with a limited survey subsample -- but make sure you do this in an intelligent way. I would suggest, for this portion, a survey sample of current students (so you can get a high rate of response) and work on a clever (even if imperfect) way to separate them into treatment and control groups for some comparability -- even if it is not ideal. Many research programs have this problem, where only users of a service can (or are willing to) answer questions. And you are interested in people who do not use the service -- or who do not even use the credit union! These are great challenges to take up and I look forward to see how you address them.

Finally -- don't worry about timeline here. This is a multi-year project: data access ALONE will take over a year; credit/financial data is the most highly protected form of personal data in the country -- higher than HIPAA/health data -- and banks don't go around giving it out.

**Comment by BeaLeydier** [comment]
Hi team

Great subject and I love the idea of using a GU sample/data for the research itself. 

I agree with Ben's comments above, and have a couple things to add to help you refine your project : 

1. Can you expand the outcomes and the question to learn something more? If I understand your design correctly, the main question you would be answering is along the lines of "are people more likely to make payments on time when payments are automated" and the response to that is both a little obvious and not super interesting. In what way the automation of payments improve participants livelihoods or other more general outcomes? For example, you could be looking at other individual outcomes like time use, mental health, which could be affected by either the potential improvement of the credit score or the not having to remember to make payments aspect of the program. Also, how does it affect the credit union's business model? Are there local "market" effects that affect other credit union clients even if they don't participate in this scheme (this may be something you are hinting to when talking about an oversaturated market)? etc

2. Assuming the program has already been implemented, how are you going to obtain survey data prior its implementation? (I don't think that data has been collected already?)

3. One thing that comes to mind also conceptually (though it may not apply given the context of your implementation, I am not sure), is because you are claiming CBLs are not effective at building up credit scores (which is something you can document/support a little more in your background by quoting literature), but automation could be, is whether a program that doesn't involve CBLs but adds an automation component to other typical credit lines would be similarly effective; and if yes, why, and if not why not.

Cheers!

**Comment by JasmineAdams** [comment]
I appreciate the feedback on this! The methodology was a little underdeveloped because we were initially misinformed about when the program launched. The credit union marketed the program as if it launched in 2020 (hence why we thought a regression discontinuity design would work well here, especially if only current students were eligible). I later saw on their social media page that the program began well before 2020. They called it "new" because it's now offered in partnership with the university, which will now cover the sign-up cost of the $1000 loan option for up to 1000 current GU students annually (the unsubsidized cost is $30). This makes a discontinuity design less feasible since different outcomes between C'19 and C'20 would indicate the effect of waiving the cost of the program rather than offering it in general.

After giving it some thought, I think an RCT could actually work here (or something close to it) since alumni, grad students, and their immediate families are also eligible for the program (just without the subsidy, which our hypothetical funders could cover). I don't think too many non-undergrads know about the program or even the credit union (I only learned about it after googling what a credit union was and stumbling upon a list of credit unions in DC.). We couldn't design a perfect RCT but we could finesse something that's close enough. We proposed sampling 200 people since many people's primary accounts are with other banks, so most of our data collection would be through more concerted efforts. 

To clarify outcomes, we were thinking more along the lines of does the program have a sustained, positive impact on scores, or do scores converge closer to the comparison group's overtime? Or does it end up replacing other common credit-building tools? I agree we should measure other supplementary outcomes in addition to financial data. We can also flesh out issues with other programs as well, given the self-financing aspect of the program is also part of why it works so well and is equally unique.

**Comment by bbdaniels** [comment]
Hi team,

Thanks for submitting the concept note. The main missing piece here is in the methodology; the rest of the background and motivation are fine. Overall, you need to provide substantially more detail about what you would plan to do here, and this means getting specific. For example, how many individuals will you recruit? Will you have access to information about them from the credit union? How many do you think will qualify vs not qualify for the study?

Then, you need to be very clear about what your treatment and control groups are, what they will each receive, and how many people are in each. The control group has already received the survey, so are they supposed to just remain unaware of the CBL program? If so, how are you measuring their outcomes? "Accessing FICO scores" is listed; is this the main outcome? If so, are you planning to effectively collect social security numbers and then use them to access sensitive data? This is possible -- but it is highly sensitive and the two sentences you have on that don't really cover the importance of handling this correctly and how you will calculate outcomes from there. Also, keep in mind that as you develop many outcomes you may want to think about multiple test corrections in your design -- see Bonferroni and Benjamini-Hochberg type discussions on that.

If I understand correctly overall, the idea is to have a treatment group with a fully subsidized and repaid CBL loan. Since this is an intention-to-treat design, for the final submission you should form clear ideas about the data generating process -- here, especially, in the take-up rates and contamination rates (that is, people from treatment who don't take the loan, and people from control who do); as well as survey non-response. You should explore those potential issues in the Week 10 assignment so that it is ready for the final submission. This is a fine start and a creative idea; once you guild out these specifics it will work. Keep it up!
